release-notes/opensearch-dashboards.release-notes-1.3.1.md

```diff
@@ -0,0 +1,65 @@
+## Version 1.3.1 Release Notes
+
+- **[Branding] Allow for SSL setup failures (#1414)(#1417)**
+
+  [Kawika Avilla](mailto:kavilla414@gmail.com)
+
+  Setup HTTP Agent in the render portion when it did not need to be
+  it just needed a one time setup for the life time of the server.
+
+  Also if this fails to read the keys then it would fail. But it's
+  only used for custom branding. We shouldn't failed for custom branding
+  just rely on default branding.
```

Comment:
I am not sure what `this` is referring to. Can we add some context to this please?